clients/client-java/src/test/java/org/apache/gravitino/client/integration/test/authorization/AccessControlIT.java

```diff
@@ -489,7 +489,18 @@ void testManageRolePermissions() {
             metalake.grantPrivilegesToRole(
                 roleName,
                 duplicatedCatalog,
-                Lists.newArrayList(Privileges.SelectTable.allow(), Privileges.SelectTable.deny())));
+                Lists.newArrayList(Privileges.ReadFileset.allow(), Privileges.ReadFileset.deny())));
```

Comment:
I think maybe we can use `Set<Privilge>` in the `grantPrivilegesToRole(..., ..., Set<Privilge> privileges)` ?

Reply:
Modified.

Reply:
I modified the implement of server. If we want to change the client side, we should deprecate the API and add new API.

Reply:
All changed.